tools/ThermoRawFileParser/thermo_converter.xml

```diff
@@ -1,60 +1,227 @@
-<tool id="thermo_raw_file_converter" name="Thermo" version="1.1.2">
+<tool id="thermo_raw_file_converter" name="Thermo" version="1.1.10">
     <description>RAW file converter</description>
     <requirements>
-        <requirement type="package" version="1.1.2">ThermoRawFileParser</requirement>
+        <requirement type="package" version="1.1.10">ThermoRawFileParser</requirement>
     </requirements>
     <command>
 <![CDATA[
+#import re
 
-ln -s '$input' ./input.raw &&
+#set $temp_stderr = "thermo_converter_stderr"
 
-ThermoRawFileParser.sh
-    -i=input.raw
-    -o=./
+echo "" > $temp_stderr &&
+
+mkdir ./raws_folder &&
+mkdir ./output_folder &&
+#for $input_raw in $input:
+    #if len($input) > 1
+        #set $input_name = re.sub('[^\w\-\.]', '_',$input_raw.element_identifier.split('/')[-1].replace(".raw", "") + ".raw")
+        ln -s -f '${input_raw}' './raws_folder/${input_name}' &&
+    #else:
+        ln -s -f '${input_raw}' './raws_folder/input.raw' &&
+    #end if
+#end for
+
+(ThermoRawFileParser.sh
+    -d=./raws_folder
+    -o=./output_folder
     -f=$output_format
-    #if $metadata:
-        --metadata=1
+    #if $gzip_boolean == 1:
+    -g
+    #end if
+    #if $output_metadata_selector != "off":
+        --metadata="${output_metadata_selector}"
+    #end if
+    #if $zlib_boolean == 0:
```

Comment:
These are booleans you can use `truevalue="-z" falsevalue=""` and here you can save the if condition and just wite `$zlib_boolean`, which will evaluate to an empty string or `-z`